patches/components-password_manager-core-browser-affiliation-hash_affiliation_fetcher.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc b/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
+index 842a3fe44617d818a3424eb75383c5527681a78a..7bd577b4e0acebc2e96fb61ec33aeeae57405760 100644
+--- a/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
++++ b/components/password_manager/core/browser/affiliation/hash_affiliation_fetcher.cc
+@@ -54,6 +54,7 @@ HashAffiliationFetcher::~HashAffiliationFetcher() = default;
+ void HashAffiliationFetcher::StartRequest(
+     const std::vector<FacetURI>& facet_uris,
+     RequestInfo request_info) {
++  BRAVE_HASH_AFFILIATION_FETCHER_START_REQUEST
```

Comment:
it might be actually possible to subclass `HashAffiliationFetcher` and override `StartRequest` to remove this patch.

Reply:
Indeed. I've revised the PR.